bin/run_app.py

```diff
@@ -1,38 +1,63 @@
 __author__ = 'mwham'
-import sys
 import os
-import multiprocessing
+import argparse
 import logging
 import logging.config
 import logging.handlers
-sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-import reporting_app
-import rest_api
-from config import rest_config, reporting_app_config
+
+watched_files = [
+    os.path.join(os.path.dirname(__file__), '..', 'etc', 'schema.yaml')
```

Comment:
Does this mean only the schema is watched ie: if the schema does not change then the reload won't happen?